### PR TITLE
demo_solo12: Fix demo and run calibration procedure by default

### PR DESCRIPTION
**Pull request by jviereck**

### CODE REVIEW

File: demos/demo_solo12.cpp [hunk 1]
```diff
@@ -61,25 +65,36 @@ static THREAD_FUNCTION_RETURN_TYPE control_loop(void* args)
...
 
     rt_printf("control loop started \n");
 
+    rt_printf("start calibration \n");
+
+    // Calibrate the robot.
+    blmc_robots::Vector12d joint_index_to_zero =
+        thread_data_ptr->joint_index_to_zero;
+
+    thread_data_ptr->robot->calibrate(joint_index_to_zero);
```
Comment (MaximilienNaveau) [inline demos/demo_solo12.cpp:79]: should we pause here? to allow the user to understand that the calibration is going to be played?

**Review by MaximilienNaveau — COMMENTED** [review]
Thanks for the PR!

File: demos/demo_solo12.cpp (hunk 1 above)
Comment (jviereck) [reply demos/demo_solo12.cpp:79]: I rather like it without but can see that it can be confusing for the user. I will add it again.

Comment (jviereck) [reply demos/demo_solo12.cpp:79]: Done.

@MaximilienNaveau , can you double check and if things are okay now approve the PR?

**Pull request merged**